deployment/terraform/variables.tf

```diff
@@ -301,6 +301,15 @@ variable "franklin_migrations_memory" {
   type = number
 }
 
+variable "aws_azavea_account_id" {
+  type = string
+}
```

Comment:
Set to the RF account ID, where we run Jenkins.

Reply:
The name here feels misleading. When I looked at the deployment docs up above, I thought this referred to our root account ID.

Reply:
Do you think that `aws_raster_foundry_account_id` or a more generic `aws_deploy_account_id` would be clearer?